randovania/games/dread/json_data/Artaria.txt

```diff
@@ -1288,10 +1295,9 @@ Extra - asset_id: collision_camera_011
               All of the following:
                   Gravity Suit
                   Simple IBJ or Use Spin Boost
+              Movement (Intermediate) and Lay Cross Comb
   > Tile Group (BOMB) 3
-      Any of the following:
-          Space Jump or Infinite Bomb Jump (Intermediate)
-          Gravity Suit and Simple IBJ
+      Space Jump or Simple IBJ
```

Comment:
Could add PB + Speed Booster

Reply:
Just speed. PBs would not be considered cause it's covered by the tile block node weakness, and the blocks could be randomized to SA or Missile which would mean PBs would just be wasted.